en/sim_gazebo_gz/tools_avl_automation.md

```diff
@@ -0,0 +1,162 @@
+# Advanced Lift Drag (AVL) Automation Tool
+
+The Gazebo [Advanced Plane](../sim_gazebo_gz/vehicles.md#advanced-plane) vehicle model uses the _Advanced Lift Drag_ plugin to model vehicle lift and drag behaviour.
+This tool allows you to calculate the parameters required to create a _Advanced Lift Drag_ plugin for your own particular vehicle.
```

Comment:
FYI This is a more direct explanation of purpose - you use this to customize advanced plane for your purpose